### PR TITLE
- Refactor tests/ & app/ folders to use let & const and destructuring…

### PR DESCRIPTION
**Pull request by Mawaheb**
This is a first step of WIP that covers [#360](https://github.com/emberjs/ember-inspector/issues/360)

In this PR, the contents of App/ and Test/ folders are refactored to use ES6's `let` & `const`  and use destructuring to import Ember classes.

### CODE REVIEW

**Comment by locks** [comment]
:+1:

File: app/adapters/websocket.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import Ember from "ember";
 import BasicAdapter from "./basic";
+const { computed } = Ember;
 
-var computed = Ember.computed;
-
-var WebsocketAdapter = BasicAdapter.extend({
+let WebsocketAdapter = BasicAdapter.extend({
```
Comment (teddyzeenny) [inline app/adapters/websocket.js]: We should just `export default BasicAdapter.extend`

File: app/controllers/iframes.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
 import Ember from "ember";
-var ArrayController = Ember.ArrayController;
-var alias = Ember.computed.alias;
-var mapComputed = Ember.computed.map;
-var run = Ember.run;
+const { ArrayController, computed, run } = Ember;
+const { alias, map } = computed;
+const mapComputed = map;
```
Comment (teddyzeenny) [inline app/controllers/iframes.js]: This should either be `const { alias, map: mapComputed } = computed` or we just use `map` in the file below.

File: app/controllers/promise-item.js [hunk 3]
```diff
@@ -1,16 +1,14 @@
 import Ember from "ember";
-var COLOR_MAP = {
+const { computed } = Ember;
+const { alias, notEmpty, empty, gt, equal } = computed;
+const computedEqual = equal;
```
Comment (teddyzeenny) [inline app/controllers/promise-item.js]: Let's just use `equal`, no need to have two names for the same function.

File: app/controllers/render-item.js [hunk 4]
```diff
@@ -1,10 +1,9 @@
...
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
-var ObjectController = Ember.ObjectController;
-var gt = Ember.computed.gt;
-var oneWay = Ember.computed.oneWay;
-var isEmpty = Ember.isEmpty;
-var runOnce = Ember.run.once;
+const { ObjectController, computed, isEmpty, run} = Ember;
+const { gt, oneWay } = computed;
+const { once } = run;
+const runOnce = once;
```
Comment (teddyzeenny) [inline app/controllers/render-item.js]: `once` should be enough.

File: app/models/promise.js [hunk 5]
```diff
@@ -1,12 +1,9 @@
...
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
 import computed from 'ember-new-computed';
-var typeOf = Ember.typeOf;
-var or = Ember.computed.or;
-var equal = Ember.computed.equal;
-var not = Ember.computed.not;
+const { typeOf, computed: { or, equal, not } } = Ember;
 
-var dateComputed = function() {
+let dateComputed = function() {
```
Comment (teddyzeenny) [inline app/models/promise.js]: This function is better as `const` or just `function dateComputed()`

File: tests/ember_debug/container_debug_test.js [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
-var emberA = Ember.A;
+let emberA = Ember.A;
 
-var EmberDebug;
-var port, name, message;
-var run = Ember.run;
-var App;
+let EmberDebug;
+let port, name, message;
+let run = Ember.run;
```
Comment (teddyzeenny) [inline tests/ember_debug/container_debug_test.js]: `const { run, A: emberA } = Ember;`

File: tests/ember_debug/ember_debug_test.js [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
 
-var EmberDebug;
-var name, port, adapter;
+let EmberDebug;
+let port, adapter;
 /* jshint ignore:start */
-var run = Ember.run;
-var App;
-var EmberInspector;
+let run = Ember.run;
```
Comment (teddyzeenny) [inline tests/ember_debug/ember_debug_test.js]: `const { run, Application, Object: emberObject } = Ember`;

File: tests/ember_debug/render_debug_test.js [hunk 8]
```diff
@@ -3,10 +3,10 @@ import Ember from "ember";
...
 
 /* globals require */
-var EmberDebug = require('ember-debug/main')["default"];
-var port;
-var App, run = Ember.run;
-var compile = Ember.Handlebars.compile;
+const EmberDebug = require('ember-debug/main')["default"];
+let port;
+let App, run = Ember.run;
+const compile = Ember.Handlebars.compile;
```
Comment (teddyzeenny) [inline tests/ember_debug/render_debug_test.js]: `const { run, Handlebars: { compile } } = Ember`;

**Comment by teddyzeenny** [comment]
@Mawaheb looks great!! Left a few minor comments.

**Comment by Mawaheb** [comment]
@teddyzeenny  Thanks for the feedback, I will make the changes and push again :)

**Comment by Mawaheb** [comment]
Done!, Please let me know if you have any notes.

**Comment by teddyzeenny** [comment]
Nice!

**Pull request merged**